docs/document/content/dev-manual/configuration.cn.md

```diff
@@ -24,6 +24,8 @@ chapter = true
 | ShardingRuleBuilder                               | 用于将分片用户配置转化为分片规则对象                        |
 | EncryptRuleBuilder                                | 用于将加密用户配置转化为加密规则对象                        |
 | ShadowRuleBuilder                                 | 用于将影子库用户配置转化为影子库规则对象                     |
+| AlgorithmProvidedShadowRuleBuilder                | 用于将基于算法的影子库用户配置转化为影子库规则对象             |
```

Comment:
I suggest putting all AlgorithmProvided rules at the top of the table

Reply:
> I suggest putting all AlgorithmProvided rules at the top of the table

Done, please review, thank you.